### PR TITLE
17869 - AUTH-WEB - Added CFS Account Number

### PR DESCRIPTION
**Pull request by AbrahamRostampoor**
*Issue #:*
https://github.com/bcgov/entity/issues/17869

*Description of changes:*
Created a new branch. Added CFS account number to auth-web payment page.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the sbc-auth license (Apache 2.0).

### CODE REVIEW

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://bcregistry-account-dev--pr-2608-6osobl5x.web.app

File: auth-web/src/components/pay/PayWithOnlineBanking.vue [hunk 1]
```diff
@@ -47,7 +47,7 @@
         <li>Sign in to your financial institution's online banking website or app</li>
         <li>Go to your financial institution's bill payment page</li>
         <li>Enter "BC Registries and Online Services" as payee</li>
-        <li>Enter BC Registries and Online Services account number</li>
+        <li>Enter BC Registries and Online Services account number: {{ cfsAccountId }}</li>
```
Comment (Jxio) [inline auth-web/src/components/pay/PayWithOnlineBanking.vue:50]: I think color for cfsaccount # is #495057 at design doc

File: auth-web/src/components/pay/PayWithOnlineBanking.vue [hunk 2]
```diff
@@ -47,7 +47,7 @@
         <li>Sign in to your financial institution's online banking website or app</li>
         <li>Go to your financial institution's bill payment page</li>
         <li>Enter "BC Registries and Online Services" as payee</li>
```
Comment (Jxio) [inline auth-web/src/components/pay/PayWithOnlineBanking.vue:49]: "BC Registries and Online Services" color seems need to update, check with Ethan.

**Comment by Jxio** [comment]
update package version

**Review by seeker25 — CHANGES_REQUESTED** [review]
image

**Comment by seeker25** [comment]
package + package-lock update please

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://bcregistry-account-dev--pr-2608-6osobl5x.web.app

**Comment by ethantspitt** [comment]
Is it possible to attach related screenshot? using the bcbrun link seems not showing the new changes:
Screenshot 2023-10-18 at 11 20 15 AM

**Comment by AbrahamRostampoor** [comment]
> Is it possible to attach related screenshot? using the bcbrun link seems not showing the new changes: Screenshot 2023-10-18 at 11 20 15 AM

> Yes for sure Ethan. Here is the screenshot I got. please let me know if I should take another one.
MicrosoftTeams-image

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://bcregistry-account-dev--pr-2608-6osobl5x.web.app

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://bcregistry-account-dev--pr-2608-6osobl5x.web.app

**Review by Jxio — APPROVED** [review]
LGTM!

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://bcregistry-account-dev--pr-2608-6osobl5x.web.app

**Review by seeker25 — APPROVED** [review]

**Pull request merged**